pkg/apis/extensions/v1alpha1/types_controlplane.go

```diff
@@ -67,6 +67,8 @@ type ControlPlaneSpec struct {
 	DefaultSpec `json:",inline"`
 	// Purpose contains the data if a cloud provider needs additional components in order to expose the control plane.
 	// This field is immutable.
+	// Deprecated: This field will be removed in a future release.
+	// The value "exposure" is no longer used since the enablement of SNI, and the value "normal" is redundant.
```

Comment:
We could add a single TODO for the removal of the field so that it is clear who will take care of the removal. 😅 

```suggestion
	// The value "exposure" is no longer used since the enablement of SNI, and the value "normal" is redundant.
	// TODO(theoddora): Remove the field in v1.123.
```